src/m365/planner/commands/task/task-list.ts

```diff
@@ -48,14 +50,22 @@ class PlannerTaskListCommand extends GraphCommand {
   }
 
   public commandAction(logger: Logger, args: CommandArgs, cb: () => void): void {
+    if (args.options.planName) {
+      args.options.planTitle = args.options.planName;
+
+      if (this.verbose) {
```

Comment:
Let's not put deprecation notices behind verbose switches. How are you going to notice they are deprecated otherwise? Verbose is not in the regular workflow of many users.